### PR TITLE
Issues/#153 eqb folder

### PR DESCRIPTION
**Pull request by r-soltani**
- added a new decoding script.
- fixed typo

### CODE REVIEW

File: eqb/scripts/hash160-sha3.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import hashlib
 import sha3
 
-# This is a handy script to generate a Bitcoin HASH160 which is RIPEMD160(SHA256(input)))
+
+# This is a handy script to generate a Equibit HASH160 which is RIPEMD160(SHA3(input)))
```
Comment (macterra) [inline eqb/scripts/hash160-sha3.py:5]: Alternatively:
```
k = hashlib.new('sha3_256')
```

@ealefad is this more portable between python 2 and 3?

Comment (ealefad) [reply eqb/scripts/hash160-sha3.py:5]: This is exactly how it is being used in functional tests, however `hashlib` doesn't support `sha3` before it was added to `python 3.6` (I couldn't install `hashlib` with `sha3` with pip on python 2). But we don't need python 2, right? Let's stick to 3 everywhere.

**Review by ealefad — APPROVED** [review]

File: eqb/scripts/hash160-sha3.py (hunk 1 above)
Comment (macterra) [reply eqb/scripts/hash160-sha3.py:5]: @r-soltani said in Slack:
> I will replace k = sha3.keccak_256() with  hashlib.new('sha3_256') instead

Comment (macterra) [reply eqb/scripts/hash160-sha3.py:5]: Please don't resolve the conversation until the issue is resolved.

Comment (macterra) [reply eqb/scripts/hash160-sha3.py:5]: I just discovered that this will also work:
```
from hashlib import sha3_256
```

Comment (r-soltani) [reply eqb/scripts/hash160-sha3.py:5]: the change to the library used for sha3 will be added in the next PR. please accept this PR so team has access to the script folder structure

**Review by macterra — APPROVED** [review]
Fix deferred until next PR

File: eqb/scripts/hash160-sha3.py (hunk 1 above)
Comment (r-soltani) [reply eqb/scripts/hash160-sha3.py:5]: The changes have been added to this PR

**Pull request merged**